op-chain-ops/deployer/init.go

```diff
@@ -75,10 +75,15 @@ func Init(cfg InitConfig) error {
 	}
 
 	intent := &state.Intent{
-		L1ChainID:            cfg.L1ChainID,
-		FundDevAccounts:      true,
-		ContractsRelease:     "op-contracts/v1.6.0",
-		ContractArtifactsURL: V160ArtifactsURL,
+		L1ChainID:                  cfg.L1ChainID,
+		FundDevAccounts:            true,
+		ContractsRelease:           "op-contracts/v1.6.0",
+		ContractArtifactsURL:       V160ArtifactsURL,
+		BaseFeeVaultRecipient:      common.Address{},
+		L1FeeVaultRecipient:        common.Address{},
+		SequencerFeeVaultRecipient: common.Address{},
+		Eip1559Denominator:         50,
```

Comment:
default values taken from: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/deploy-config/mainnet.json#L39-L40

Reply:
It would be nice to only hardcode these default values in only one location, then import them in other spots